app/src/app/ui/custom-theme.scss

```diff
@@ -1,81 +1,21 @@
 @import '~@angular/material/theming';
 @import './overrides/adf-style-fixes.theme';
-@import './custom-theme-palettes';
+@import './colors';
 
-@include mat-core();
+$mat-primary-palette: mat-palette($aca-primary-blue, A100);
+$mat-accent-palette: mat-palette($aca-accent-green, A200);
+$mat-warn-palette: mat-palette($aca-warn, A100);
+
+@include mat-core($alfresco-typography);
 
 $custom-theme: mat-light-theme(
   $mat-primary-palette,
   $mat-accent-palette,
-  $mat-warn-palette
+  $mat-warn-palette,
 );
 
-$foreground: map-get($custom-theme, foreground);
```

Comment:
All variables were moved from custom-theme to `app/src/app/ui/variables/variables.scss`